### PR TITLE
fix: Ignoring CPU realtime on cgroupsv2 if set to zero

### PR DESCRIPTION
**Pull request by CheatCodeSam**
## Description
Requested to break up #3174 into two seperate issues by @YJDoc2.

This fixes an auxiliary issue of #2994, where `nerdctl` will set both `linux.resources.cpu.realtimeRuntime` and `linux.resources.cpu.realtimePeriod` to zero in the config.json despite youki trying to use cgroupsv2. This would cause a V2CpuControllerError to be thrown when the container is being created.

My solution is just to ignore them if using cgroupsv2 and they are both set to zero.

## Type of Change

- [X] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Refactoring (no functional changes)
- [ ] Performance improvement
- [ ] Test updates
- [ ] CI/CD related changes
- [ ] Other (please describe):

## Testing

- [X] Added new unit tests
- [ ] Added new integration tests
- [X] Ran existing test suite
- [X] Tested manually (please provide steps)

Once the initial runtime spec was fixed, this was able to run with no problems on nerdctl.

## Related Issues

Fixes part of  #2994.

### CODE REVIEW

**Review by utam0k — CHANGES_REQUESTED** [review]
Please add the unit test for your change 🙏

**Comment by CheatCodeSam** [comment]
Tests have been added

File: crates/libcgroups/src/v2/cpu.rs [hunk 1]
```diff
@@ -369,4 +374,36 @@ mod tests {
         let actual = fs::read_to_string(burst_file).expect("read burst file");
         assert_eq!(actual, expected.to_string());
     }
+
+    #[test]
```
Comment (utam0k) [inline crates/libcgroups/src/v2/cpu.rs:378]: Add the unit test where both values are zero.

Comment (CheatCodeSam) [reply crates/libcgroups/src/v2/cpu.rs:378]: Just set this.

**Review by utam0k — APPROVED** [review]
Thanks!
